public/css/admin.css

```diff
@@ -1,13 +1,20 @@
 /*--------------- MAILING LISTS -----------------*/
-h3 {
+
+head {
 	font-family: "Source Code Pro", monospace;
 }
```

Comment:
Not sure what you were trying to achieve here, but the `head` tag just holds metadata and isn't ever visible on a page. If you don't need this style (which it looks like you don't), then just delete it